docs/sources/features/datasources/loki.md

```diff
@@ -134,8 +134,37 @@ Note that Live Tailing relies on two Websocket connections: one between the brow
 ```
 ProxyPassMatch "^/(api/datasources/proxy/\d+/loki/api/v1/tail)" "ws://127.0.0.1:3000/$1"
 ```
+Example below shows basic NGINX proxy configuration. This example assumes that the Grafana server is available at `http://localhost:3000/`, Loki server is running locally without proxy, and your external site uses HTTPS. If you also host Loki behind NGINX proxy, then you might want to repeat the following configuration for Loki as well.
 
-> **Note:** This feature is only available in Grafana v6.3+
+In `http` section of NGINX configuration, add the following map definition:
```

Comment:
In the `http` section of NGINX configuration, add the following map definition:

Reply:
Done